Scripts/s4ap/utils/s4ap_career_utils.py

```
@@ -0,0 +1,164 @@
+            return tuple()
+        career_tracker = sim_info.career_tracker
+        if career_tracker is None:
+            return tuple()
+
+        for career in career_tracker.careers.values():
+            if include_career_callback is not None and not include_career_callback(career):
+                continue
+            yield career
+
+    @staticmethod
+    def determine_entry_level_into_career_from_user_level(career: Career, desired_user_level: int) -> Tuple[
```

Ruff (0.14.7)

Unused static method argument: `desired_user_level`

(ARG004)